src/index.ts

```diff
@@ -671,6 +671,102 @@ export interface JIT {
   extensions?: { [key: string]: string }
 }
 
+/**
+ * FormFieldIdentitySpace
+ */
+export interface FormFieldIdentitySpace {
+  id?: string
+  name?: string
+  code?: string
+}
+
+/**
+ * FormFieldConfig
+ */
+export interface FormFieldConfig {
+  formFieldId?: string
+  required?: boolean
+  label?: string
+  hint?: string
+  category?: FormFieldCategory
+  type?: FormFieldType
+  Variant?: FormFieldVariant
+  maxLength?: number
+  name?: string
+  width?: FormFieldWidth
+  identitySpace?: FormFieldIdentitySpace
+  options?: FormFieldDropdownOption[]
+}
+
+/**
+ * FormFieldDropdownOption
+ */
+export type FormFieldDropdownOption = {
+  label?: string
+  value?: string
+  translations?: { [key: string]: string }
```

Comment:
How does this work?

Reply:
pretty much like `<option value={value}>{label}</option>`
translations are `langCode: labelTranslation`

Reply:
If this is returned in the experience, then the `value` should already be translated. The `translations` like this are only used in supercargo.